File: Dataset/Leetcode/test/48/436.cpp
 class Solution {
public:
    void XXX(vector<vector<int>>& m) {
        if(!m.size() || !m[0].size()) return;
        int rows=m.size(); int cols = m[0].size();
        for(int i = 0;i<rows;i ++)
        {
            for(int j=0;j<i;j++)
            {
                swap(m[i][j],m[j][i]);
            }
        }
        for(int i=0;i<rows;i++)
        {
            int l = 0; int r = m[i].size() - 1;
            while(l < r)
            {
                swap(m[i][l++],m[i][r--]);
            }
        }
    }
};

File: Dataset/Leetcode/train/7/436.py
 class Solution:
    def XXX(self, x: int) -> int:
        t = list(str(x))
        negative = False
        if t[0] == "-":
            negative = True
            t = t[1:]
        y = int("".join(XXXd(t)))
        if y < -2 ** 31 or y > 2 ** 31 - 1:
            y = 0
        return y if not negative else -y

